Plots/Streamlines/NCL_stream_1.py

```diff
@@ -0,0 +1,89 @@
+"""
+NCL_stream_1.py
+===============
+Basic streamlines drawing
+
+Concepts illustrated:
+   - Drawing a black-and-white streamline plot over a map
+
+Replicates the NCL script: https://www.ncl.ucar.edu/Applications/Scripts/stream_1.ncl
+
+Note: The streamlines generated in this plot approximate, but do not exactly
+match those generated by the NCL version. This is primarily because the 
+seed point generation algorithm used by NCL is opaque.
+"""
+
+################################################################################
+#
+# import modules
+#
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+import geocat.viz as gcv
+import geocat.datafiles
+
+
+################################################################################
+#
+# open data file and extract variables. We extract a 2D horzontal slice from
+# the first time step of the 3D U and V variables at the at the bottom level.
+# 
+ds = xr.open_dataset(geocat.datafiles.get('netcdf_files/uvt.nc'))
+U = ds.U.isel(time=0, lev=0)
+V = ds.V.isel(time=0, lev=0)
+
+
+################################################################################
+#
+#
+plt.rcParams['figure.figsize'] = [16, 8]
```

Comment:
I find 

```suggestion
fig.set_size_inches((16,8))
```

much more clear.

It has to come after `fig=plt.figure()`

Reply:
Agreed! And done.